src/binaryManager.ts

```diff
@@ -19,37 +21,42 @@ function getExtensionMirrordPath(): Uri {
 
 /**
  * Tries to find local mirrord in path or in extension storage.
+ * @param version If specified, then the version of the binary is checked and matched path is returned if it matches.
+ * @returns Path to mirrord binary or null if not found
  */
 export async function getLocalMirrordBinary(version?: string): Promise<string | null> {
     try {
-        const mirrordPath = await which("mirrord");
+        const mirrordPath = getExtensionMirrordPath();
+        await workspace.fs.stat(mirrordPath);
```

Comment:
Why is this necessary?

Reply:
i did not add this, existed before, i guess it just makes sure that it exists and the exception is caugght